queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/RequestCommands/PosReceiptCommand.cs

```diff
@@ -15,28 +17,37 @@ public class PosReceiptCommand : RequestCommand
         private readonly ICountrySpecificQueueRepository _countrySpecificQueueRepository;
         private readonly ICountrySpecificSettings _countryspecificSettings;
         private readonly IConfigurationRepository _configurationRepository;
+        private readonly IMiddlewareQueueItemRepository _middlewareQueueItemRepository;
+        private readonly SignatureItemFactoryDEFAULT _signatureItemFactory;
 
-        public PosReceiptCommand(IConfigurationRepository configurationRepository, ICountrySpecificSettings countrySpecificSettings)
+        public PosReceiptCommand(IConfigurationRepository configurationRepository, ICountrySpecificSettings countrySpecificSettings, IQueueItemRepository queueItemRepository, IMiddlewareQueueItemRepository middlewareQueueItemRepository)
```

Comment:
don't need these changes I think 🤔

Reply:
The lines
```cs
        private readonly ICountrySpecificSettings _countryspecificSettings;
        private readonly IConfigurationRepository _configurationRepository;
        private readonly IMiddlewareQueueItemRepository _middlewareQueueItemRepository;
```
can be removed here and from the constructor as they seem to be unused 🤔